Gordon360/Controllers/RecIM/ParticipantsController.cs

```diff
@@ -81,11 +81,10 @@ public async Task<ActionResult<ParticipantExtendedViewModel>> SetParticipantAdmi
 
         [HttpPatch]
         [Route("{username}/emails")]
-        public async Task<ActionResult<ParticipantExtendedViewModel>> UpdateParticipant(string username, [FromBody] bool allowEmails)
+        [StateYourBusiness(operation = Operation.UPDATE, resource = Resource.RECIM_PARTICIPANT)]
+        public async Task<ActionResult<ParticipantExtendedViewModel>> SetParticipantAllowEmails(string username, [FromBody] bool allowEmails)
```

Comment:
```suggestion
        public async Task<ActionResult<ParticipantExtendedViewModel>> SetParticipantAllowEmailsAsync(string username, [FromBody] bool allowEmails)
```

Reply:
Should the route also require the "Async" tag? I don't think we do that anywhere else in RecIM. I could make that a cleanup change.

Reply:
👍🏼 resolved in person. This is something todo